modules/gdscript/language_server/gdscript_workspace.cpp

```diff
@@ -396,24 +396,26 @@ void GDScriptWorkspace::_get_owners(EditorFileSystemDirectory *efsd, String p_pa
 		}
 		if (!found)
 			continue;
+		if (efsd->get_file_path(i).get_extension().to_lower() != "tscn")
+			continue;
 
 		owners.push_back(efsd->get_file_path(i));
 	}
 }
 
-Node *GDScriptWorkspace::_get_owner_node(String p_path) {
-	Node *owner_node = NULL;
-	List<String> owners;
+Node *GDScriptWorkspace::_get_owner_scene_node(String p_path) {
+	Node *owner_scene_node = NULL;
+	List<String> owner_scenes;
 
-	_get_owners(EditorFileSystem::get_singleton()->get_filesystem(), p_path, owners);
+	_get_owner_scenes(EditorFileSystem::get_singleton()->get_filesystem(), p_path, owner_scenes);
 
-	if (owners.size() > 0) {
-		NodePath owner_path = owners[0];
-		Ref<PackedScene> owner_res = ResourceLoader::load(owner_path);
-		owner_node = owner_res->instance(PackedScene::GEN_EDIT_STATE_DISABLED);
+	if (owner_scenes.size() > 0) {
+		NodePath owner_scene_path = owner_scenes[0];
+		Ref<PackedScene> owner_scene_res = ResourceLoader::load(owner_scene_path);
```

Comment:
Check `owner_scene_res ` is valid here.
If it is not valid check next path in the vector you filled.

Reply:
@Geequlim Could you help me with this one? How can I do this check, is there any props in the PackedScene I could look into?

Reply:
Can't we do something like this:
https://github.com/godotengine/godot/blob/master/main/main.cpp#L1460

Reply:
Ok nvm, I figured out gonna send a new commit soon